cognite/client/data_classes/workflows.py

```diff
@@ -535,6 +553,7 @@ def __init__(
         start_time: int | None = None,
         end_time: int | None = None,
         reason_for_incompletion: str | None = None,
+        type: str | None = None,
```

Comment:
Same here, type is already kept in the class type.

Reply:
Same solution her, I think just a property will give you the same connivence without relying on the user of the class to ensure they input a type that is consistent with the `output` parameter.